src/pages/Exercise1.tsx

```diff
@@ -38,13 +46,18 @@ const Exercise1: FC = () => {
                       { id: 3, name: '12', displayName: '12' },
                       { id: 4, name: '13', displayName: '13' },
                     ]}
+                    correctAnswer={correctAnswer}
+                    setCorrectAnswer={setCorrectAnswer}
+                    index={0}
                   />
                 </span>
               </p>
             </Question>
             <Question
-              correct="Correct! Now Pipi needs to..."
-              incorrect="Not quite. Think about ... Please try again."
+              correct="Correct! Pipi needs to move from address 3 to address 11. Now Pipi needs to go to a different box."
```

Comment:
Pipi starts at address 2, so 11-2 should be 9 addresses. The correctAnswer is wrong here